packages/pigeon/lib/java_generator.dart

```diff
@@ -421,13 +437,15 @@ class JavaGenerator extends StructuredGenerator<JavaOptions> {
         indent.scoped('{', '}', () {
           const String channel = 'channel';
           indent.writeln('BasicMessageChannel<Object> $channel =');
-          indent.inc();
-          indent.inc();
-          indent.writeln(
-              'new BasicMessageChannel<>(binaryMessenger, "$channelName", getCodec());');
-          indent.dec();
-          indent.dec();
-          indent.write('$channel.send($sendArgument, channelReply -> ');
+          indent.inc(2);
```

Comment:
Would this series of manual inc/dec calls be easier to manage as `scoped` with empty begin/ends and a `nestCount` of the right value?

Reply:
Will update to `nest` as mentioned in offline chat